docs/examples/regenerate-index.py

```diff
@@ -2,14 +2,19 @@
 
 import json
 import os
+from sys import argv, exit
 
-def main():
-    dir = os.path.dirname(os.path.realpath(__file__))
-    viewconfs_dir = os.path.join(dir, 'viewconfs')
-    print(json.dumps(
-        sorted(os.listdir(viewconfs_dir)),
-        indent=2
-    ))
+def main(dirs):
+    not_dirs = [f for f in dirs if not os.path.isdir(f)]
```

Comment:
Would you mind adding a docstring and an error message in case the correct arguments aren't passed? This would be helpful in documenting what exactly this function does and how it can be used. I like `argparse` but there may be other packages that can do this.

Reply:
got rid of this python script